operator/builtin/input/file/util_test.go

```diff
@@ -122,7 +122,7 @@ func waitForOne(t *testing.T, c chan *entry.Entry) *entry.Entry {
 	select {
 	case e := <-c:
 		return e
-	case <-time.After(time.Second):
+	case <-time.After(5 * time.Second):
```

Comment:
Can we revert this (and the other occurrences)?

Reply:
what's wrong with it?

Reply:
I think it is necessary, maybe not all the way to 5 secs, but I am suspecting the windows server on github action is not fast enough. so when it is executing all the tests in parallel, it fails to complete the poll cycle.